guides/source/active_job_basics.md

```diff
@@ -122,7 +122,11 @@ GuestsCleanupJob.set(wait: 1.week).perform_later(guest)
 GuestsCleanupJob.perform_later(guest1, guest2, filter: 'some_filter')
 ```
 
-That's it!
+NOTE: Jobs are meant to be run with the `perform_later` and
+`perform_now` wrappers: running the job's `perform` method directly is
```

Comment:
This is a pretty long sentence. What do you think about splitting it with a period instead?
```suggestion
`perform_now` wrappers. Running the job's `perform` method directly is
```